src/__tests__/events.js

```diff
@@ -295,6 +295,28 @@ test('assigns dataTransfer properties', () => {
   expect(spy.mock.calls[0][0]).toHaveProperty('dataTransfer.dropEffect', 'move')
 })
 
+test('assigns dataTransfer non-enumerable properties', () => {
+  window.DataTransfer = function DataTransfer() {}
+  const node = document.createElement('div')
+  const spy = jest.fn()
+  const item = {
```

Comment:
I think this could just be an empty object to reduce the complexity and focus on whats relevant for this testcase

Reply:
good point